packages/dendron-next-server/pages/vscode/lookup-view.tsx

```diff
@@ -0,0 +1,173 @@
+import { DendronProps } from "../../lib/types";
+import { Form, Select, Switch } from "antd";
+import {
+  DMessageSource,
+  LookupEffectTypeEnum,
+  LookupFilterTypeEnum,
+  LookupNoteTypeEnum,
+  LookupSelectionTypeEnum,
+  LookupSplitTypeEnum,
+  LookupViewMessage,
+  LookupViewMessageEnum,
+} from "@dendronhq/common-all";
+import React, { useEffect } from "react";
+import { postVSCodeMessage } from "@dendronhq/common-frontend";
+
+export default function Lookup({ ide }: DendronProps) {
```

Comment:
we shouldn't be using `dendron-next-server` for native ui. we're actually moving on removing this. this component should live in `dendron-plugin-views` under `DendronLookupPanel`